scripts/appleseedMaya/renderGlobals.py

```diff
@@ -391,10 +391,11 @@ def create(self):
         with pm.scrollLayout("appleseedScrollLayout", horizontalScrollBarThickness=0):
             with pm.columnLayout("appleseedColumnLayout", adjustableColumn=True, width=columnWidth):
                 with pm.frameLayout(label="Sampling", collapsable=True, collapse=False):
-                    with pm.columnLayout("appleseedColumnLayout", adjustableColumn=True, width=columnWidth):
+                    with pm.columnLayout("appleseedColumnLayout", adjustableColumn=False, width=columnWidth):
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Render Passes", numberOfFields=1),
+                            ui=pm.intSliderGrp(
+                                label="Render Passes", field=True, value=1, cw=(3,160), minValue=1, 
+                                fieldMinValue=1, maxValue=100, fieldMaxValue=1000000000),
```

Comment:
I suggest using the exact same hard limits as in appleseed.studio, that would be 1000000 (one million) for the number of passes.

Reply:
ok